src/lib/operators/import_csv.hpp

```diff
@@ -1,58 +0,0 @@
-#pragma once
-
-#include <memory>
-#include <optional>
-#include <string>
-#include <vector>
-
-#include "abstract_read_only_operator.hpp"
-#include "import_export/csv_meta.hpp"
-#include "utils/assert.hpp"
-
-namespace opossum {
-
-/**
- * Creates a Table with values of the parsed csv file <filename>
- * and the corresponding meta file <filename>.meta
- * For the structure of the meta csv file see export_csv.hpp
- *
- * If the parameter `tablename` is provided, the imported table is stored in the StorageManager.
- * If a table with this name already exists, it is returned and no import is performed.
- *
- * TODO(mjendruk): is this still true?
- * Note: ImportCsv does not support null values at the moment
```

Comment:
Wasn't this fixed a while ago? If yes, please make sure that the comment is no longer there.